packages/core/feature-flags/src/types.js

```diff
@@ -3,4 +3,5 @@
 export type FeatureFlags = {|
   // This feature flag mostly exists to test the feature flag system, and doesn't have any build/runtime effect
   +exampleFeature: boolean,
+  +packageKeyInvalidation: boolean,
```

Comment:
Originally I had a "description" for feature flags as part of a richer FF object export, but decided it was unnecessary - however, it would be good to document what the feature flag is for in a comment here at least.

Reply:
Done 👍